modules/pubCommonId.js

```diff
@@ -5,6 +5,10 @@
  */
 import * as utils from '../src/utils'
 import { config } from '../src/config';
+import events from '../src/events';
+import * as url from '../src/url';
+
+const CONSTANTS = require('../src/constants.json');
```

Comment:
I’d use the `import` syntax rather than `require` for `CONSTANTS`, see the userId.js file for an example we

Reply:
Done.